File: programers_Lv1/가운데 글자 가져오기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12903?language=javascript
function solution(s) {
  var answer =
    s.length % 2 === 0
      ? s.slice(Math.floor(s.length / 2) - 1, Math.floor(s.length / 2) + 1)
      : s.slice(Math.floor(s.length / 2), Math.floor(s.length / 2) + 1);
  return answer;
}

console.log(solution("abcde"));
console.log(solution("qwer"));

File: programers_Lv1/가운데 글자 가져오기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12903?language=python3
def solution(s):
    return s[len(s) // 2] if len(s) % 2 != 0 else s[len(s) // 2 - 1:len(s) // 2 + 1]

print(solution("abcde"))
print(solution("qwer"))
